packages/components/src/components/prompts/ConfirmOnDevice/index.tsx

```diff
@@ -111,61 +112,67 @@ const isStepActive = (index: number, activeStep?: number) => {
     return index < activeStep;
 };
 
-interface Props {
+export interface ConfirmOnDeviceProps {
     title: React.ReactNode;
     successText?: React.ReactNode;
     trezorModel: 1 | 2;
     steps?: number;
     activeStep?: number;
-    animated?: boolean;
-    animation?: 'SLIDE_UP' | 'SLIDE_DOWN';
+    isConfirmed?: boolean;
     onCancel?: () => void;
 }
 
-const ConfirmOnDevice = ({
+export const ConfirmOnDevice = ({
     title,
     steps,
     activeStep,
     onCancel,
     trezorModel,
     successText,
-    animated,
-    animation = 'SLIDE_UP',
-}: Props) => (
-    <Wrapper animated={animated} animation={animation} data-test="@prompts/confirm-on-device">
-        <Left>
-            <DeviceImage height="34px" trezorModel={trezorModel} />
-        </Left>
-        <Middle>
-            <Title>{title}</Title>
-            {successText && typeof steps === 'number' && activeStep && activeStep > steps ? (
-                <Success data-test="@prompts/confirm-on-device/success">{successText}</Success>
-            ) : undefined}
-            {typeof steps === 'number' && activeStep && activeStep <= steps ? (
-                <Steps>
-                    {Array.from(Array(steps).keys()).map((s, i) => (
-                        <Step
-                            key={s}
-                            isActive={isStepActive(i, activeStep)}
-                            data-test={`@prompts/confirm-on-device/step/${i}${
-                                isStepActive(i, activeStep) ? '/active' : ''
-                            }`}
-                        />
-                    ))}
-                </Steps>
-            ) : undefined}
-        </Middle>
-        <Right>
-            <CloseWrapper>
-                {onCancel && (
-                    <Close onClick={onCancel}>
-                        <Icon icon="CROSS" size={23} />
-                    </Close>
+    isConfirmed,
+}: ConfirmOnDeviceProps) => {
+    const hasSteps = steps && activeStep;
```

Comment:
Is it safe? It seems that it doesn't happen anywhere, but should `steps === 1` and `activeStep === 0` mean `hasSteps === false`?

Reply:
Agree but I'd do an undefined check so TS doesn't complain that `activeStep` `Object is possibly 'undefined'.` in comparisons below.

Reply:
cbd767025bd9279bc2d96c7ad5381c64bd6c3767